Sources/Client/Model/Channel/Channel.swift

```diff
@@ -190,6 +196,12 @@ public final class Channel: Codable {
         unreadMessageReadAtomic.set(messageRead)
         unreadCountAtomic.set(.noUnread)
     }
+    
+    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases), callback: @escaping Client.OnEvent) -> Subscription {
+        let subscription = Client.shared.subscribe(forEvents: eventTypes, channelId: cid.id, callback: callback)
```

Comment:
```suggestion
    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases), _ callback: @escaping Client.OnEvent) -> Subscription {
        let subscription = Client.shared.subscribe(forEvents: eventTypes, cid: cid, callback)
```

Reply:
I think `callback` should keep it's parameter name for clarity. Parameter name is not required when using trailing closures anyway.

Reply:
This is a code style for trailing closures. I use it everywhere.

Reply:
It leads to confusion when trailing closures are not preferred.
```swift
func trailingClosurePreferred(someArgument: Int, callback: (Int)->()) {
    callback(someArgument)
}

func trailingClosureNotPreferred(someArgument: Int, _ callback: (Int)->()) {
    callback(someArgument)
}

// 1
trailingClosurePreferred(someArgument: 1) { (num) in
    // ....
}

// 2
trailingClosurePreferred(someArgument: 1, callback: { num in
    // ....
})

// 3
trailingClosureNotPreferred(someArgument: 1) { (num) in
    // ...
}

// 4
trailingClosureNotPreferred(someArgument: 1, { num in
    // ....
})
```

1 and 3 look exactly the same but 2 and 4 look different. 2 is the preferred way. `UIView.animate` style.